python/python/tests/test_transaction.py

```diff
@@ -38,7 +38,7 @@ async def transaction_test(
     key7 = "{{{}}}:{}".format(keyslot, get_random_string(3))
     key8 = "{{{}}}:{}".format(keyslot, get_random_string(3))
     key9 = "{{{}}}:{}".format(keyslot, get_random_string(3))
-    key10 = "{{{}}}:{}".format(keyslot, get_random_string(3))  # list
+    key10 = "{{{}}}:{}".format(keyslot, get_random_string(3))  # hyper log log
```

Comment:
I wonder why it existed before

Reply:
It was from one of the previous PRs, but left unused

Reply:
#1304